github_connector/views/view_github_organization.xml

```diff
@@ -45,33 +45,35 @@ License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
                             groups="github_connector.group_github_connector_manager"/>
                 </header>
                 <sheet>
-                    <field name="image" widget="image" class="oe_avatar oe_left"/>
-                    <div class="oe_title">
-                        <h1><field name="name"/></h1>
-                        <h3 class="oe_fade">
-                            <field name="description" class="oe_inline"/>
-                        </h3>
-                    </div>
-                    <div class="oe_right oe_button_box" name="buttons">
-                        <button type="action" name="%(action_github_repository)d"
-                                class="oe_stat_button" icon="fa-folder-o"
-                                attrs="{'invisible': [('repository_qty', '=', False)]}"
-                                context="{'search_default_organization_id': active_id}">
-                            <field name="repository_qty" string="Repositories" widget="statinfo" />
+                    <div class="oe_button_box" name="buttons">
+
+                        <button type="action" name="%(action_res_partner)d"
+                                class="oe_stat_button" icon="fa-user"
+                                attrs="{'invisible': [('member_qty', '=', False)]}"
+                                context="{'search_default_organization_ids': active_id}">
+                            <field name="member_qty" string="Members" widget="statinfo" />
                         </button>
                         <button type="action" name="%(action_github_team)d"
                                 class="oe_stat_button" icon="fa-users"
                                 attrs="{'invisible': [('team_qty', '=', False)]}"
                                 context="{'search_default_organization_id': active_id}">
                             <field name="team_qty" string="Teams" widget="statinfo" />
                         </button>
-                        <button type="action" name="%(action_res_partner)d"
-                                class="oe_stat_button" icon="fa-user"
-                                attrs="{'invisible': [('member_qty', '=', False)]}"
-                                context="{'search_default_organization_ids': active_id}">
-                            <field name="member_qty" string="Members" widget="statinfo" />
+                        <button type="action" name="%(action_github_repository)d"
+                                class="oe_stat_button" icon="fa-folder-o"
+                                attrs="{'invisible': [('repository_qty', '=', False)]}"
+                                context="{'search_default_organization_id': active_id}">
+                            <field name="repository_qty" string="Repositories" widget="statinfo" />
                         </button>
                     </div>
+
+                    <field name="image" widget="image" class="oe_avatar"/>
```

Comment:
Why are you changing the side of the image?

Reply:
All other views had the avatar at the right side. I did it to standardize the form views. 
What do you think ?

Reply:
Standard Odoo put it images on the left if I'm not wrong. Can you confirm? If so, I prefer to standardize the Odoo way.

Reply:
Actually, we were confused by my screenshots of entreprise interface.
With the previous code, the avatar was forced to be on the left side on CE and EE. With the new version of the code, the avatar is placed by the CSS.
Here are the screenshot on CE version with the new code:

<img width="1679" alt="Account Budget Template - Odoo (20191112_0939)" src="https://user-images.githubusercontent.com/547282/68681147-e06c8980-0530-11ea-8fb5-142cfc766045.png">
<img width="1679" alt="account_budget_oca - Odoo (20191112_0939)" src="https://user-images.githubusercontent.com/547282/68681148-e06c8980-0530-11ea-9a4b-b01d7455343d.png">
<img width="1679" alt="numigi - Odoo (20191112_0939)" src="https://user-images.githubusercontent.com/547282/68681149-e06c8980-0530-11ea-872f-83365a8d1e88.png">

It looks standard on CE and EE now. Both world is happy.
What do you think?

Reply:
Great then, thanks for checking!

Offtopic: do you really prefer enterprise interface over `web_responsive` OCA module? We have included a lot of extra features that are not on enterprise one.

Reply:
@pedrobaeza not my choice :) Client picked EE ;)

We did some test with the new CE interface and its enhanced and it gave us some arguments to have some of our new clients on CE.
But this on is on EE, for now ;)

Reply:
Well, IMO it doesn't matter: we have enterprise customers too, but without that module installed, and installing `web_responsive` instead.